src/Uno.UI/Controls/BindableDrawerLayout.Android.cs

```diff
@@ -517,6 +518,22 @@ public Pane(SplitViewPanePlacement placement)
 			}
 		}
 
+		protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
+		{
+			// For some reason it seams Android is always measuring the Pane with a width at 100px.
+			// This behavior could break the Measure & Arrange phases where some controls may be arranged
+			// at a wrong size.
+
+			// To fix this, we simply apply the control's constraints (min+max defined sizes)
+
+			var availableSize = ViewHelper.LogicalSizeFromSpec(widthMeasureSpec, heightMeasureSpec);
+			var sizeForChildren = this.ApplySizeConstraints(availableSize).LogicalToPhysicalPixels();
+
+			base.OnMeasure(
+				(int)sizeForChildren.Width | (int)MeasureSpecMode.AtMost,
```

Comment:
There's a method for this - https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Extensions/ViewHelper.Android.cs#L335

Reply:
Oooh thanks!